charts/icm-as/templates/jobserver-deployment.yaml

```diff
@@ -18,6 +18,14 @@ apiVersion: batch.core.intershop.de/v1
 kind: ICMJob
 metadata:
   name: {{ include "icm-as.fullname" $jobValues }}-job
+  {{- include "icm-as.podData" $jobValues | nindent 2 }}
+  labels:
+    app: icm-as-job
+    release: {{ include "icm-as.fullname" . }}
```

Comment:
the "release" should not be a label. labels are "designed" to be selectors. the release is published with annotation "meta.helm.sh/release-name".

Reply:
ICM-AS contains "release" label as well.